app/src/components/label_pane.tsx

```diff
@@ -323,29 +327,22 @@ class LabelPane extends Component<Props> {
       )
 
       const numSensors = Object.keys(this.state.task.sensors).length
-      const imageOnly =
-        this.state.task.config.itemType === types.ItemTypeName.IMAGE
-      let paneControl: JSX.Element
-      if (imageOnly) {
-        paneControl = <></>
-      } else {
-        paneControl = (
-          <List>
-            <ListItem dense disableGutters>
-              {deleteButton}
-            </ListItem>
-            <ListItem dense disableGutters>
-              <div hidden={pane.hide}>{verticalSplitButton}</div>
-            </ListItem>
-            <ListItem dense disableGutters>
-              <div hidden={pane.hide}>{horizontalSplitButton}</div>
-            </ListItem>
-            <ListItem dense disableGutters>
-              {visibilityButton}
-            </ListItem>
-          </List>
-        )
-      }
+      const paneControl = (
```

Comment:
These elements will appear when it is a 2d image labeling project. We should not show them under this scenerios